### ISSUE TITLE
Host is not auto-detected

### ISSUE THREAD

**Issue by adamralph** [adamralph/bullseye#841]
## Version(s)

4.2.0

## To reproduce

Steps to reproduce the behaviour:

1. Run some Bullseye targets in a console specifying the `--verbose` option.

## Expected behaviour

The output contains `Host: Console (detected)`.

## Actual behaviour

The output contains `Host: AppVeyor (forced)`.

## Workarounds

Always specify the host:

```text
  --appveyor                 Force AppVeyor mode (normally auto-detected)
  --console                  Force console mode (normally auto-detected)
  --github-actions           Force GitHub Actions mode (normally auto-detected)
  --gitlab-ci                Force GitLab CI mode (normally auto-detected)
  --teamcity                 Force TeamCity mode (normally auto-detected)
  --travis                   Force Travis CI mode (normally auto-detected)
```

## Additional context

See https://github.com/aaubry/YamlDotNet/pull/735#issuecomment-1335947268.

**Comment by adamralph** [comment]
fixed by #843

**Comment by adamralph** [comment]
released in [4.2.1-rc.1](https://www.nuget.org/packages/Bullseye/4.2.1-rc.1)
